<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1040</width>
    <height>626</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <layout class="QGridLayout" name="gridLayout_3">
    <item row="0" column="0">
     <widget class="QTabWidget" name="tabWidget">
      <property name="currentIndex">
       <number>0</number>
      </property>
      <widget class="QWidget" name="tab">
       <attribute name="title">
        <string>ORGANIZADOR</string>
       </attribute>
       <widget class="QLabel" name="label_12">
        <property name="geometry">
         <rect>
          <x>140</x>
          <y>70</y>
          <width>741</width>
          <height>71</height>
         </rect>
        </property>
        <property name="font">
         <font>
          <pointsize>28</pointsize>
         </font>
        </property>
        <property name="text">
         <string>ORGANIZADOR DE ARCHIVOS</string>
        </property>
        <property name="alignment">
         <set>Qt::AlignCenter</set>
        </property>
       </widget>
       <widget class="QLabel" name="label_13">
        <property name="geometry">
         <rect>
          <x>90</x>
          <y>390</y>
          <width>141</width>
          <height>71</height>
         </rect>
        </property>
        <property name="font">
         <font>
          <family>Eras Medium ITC</family>
          <pointsize>24</pointsize>
         </font>
        </property>
        <property name="text">
         <string>BY: DC</string>
        </property>
        <property name="alignment">
         <set>Qt::AlignCenter</set>
        </property>
       </widget>
      </widget>
      <widget class="QWidget" name="tabCambiarNombres">
       <attribute name="title">
        <string>Cambiar Nombres</string>
       </attribute>
       <widget class="QGroupBox" name="groupBox_2">
        <property name="geometry">
         <rect>
          <x>20</x>
          <y>10</y>
          <width>581</width>
          <height>451</height>
         </rect>
        </property>
        <property name="title">
         <string>Cambiar nombre de archivos seleccionados</string>
        </property>
        <widget class="QPlainTextEdit" name="archivosSeleccionados_cambiarNombre_plainTextEdit">
         <property name="geometry">
          <rect>
           <x>10</x>
           <y>30</y>
           <width>221</width>
           <height>401</height>
          </rect>
         </property>
        </widget>
        <widget class="QToolButton" name="seleccionarArchivos_cambiarNombre_toolButton">
         <property name="geometry">
          <rect>
           <x>490</x>
           <y>30</y>
           <width>25</width>
           <height>19</height>
          </rect>
         </property>
         <property name="text">
          <string>...</string>
         </property>
        </widget>
        <widget class="QLabel" name="label_4">
         <property name="geometry">
          <rect>
           <x>270</x>
           <y>30</y>
           <width>131</width>
           <height>16</height>
          </rect>
         </property>
         <property name="text">
          <string>Seleccionar archivos</string>
         </property>
        </widget>
        <widget class="QLabel" name="label_5">
         <property name="geometry">
          <rect>
           <x>270</x>
           <y>100</y>
           <width>181</width>
           <height>16</height>
          </rect>
         </property>
         <property name="text">
          <string>Texto que ira al prinicipio del archivo:</string>
         </property>
        </widget>
        <widget class="QLineEdit" name="textoNuevoArchivos_cambiarNombre_lineEdit">
         <property name="geometry">
          <rect>
           <x>270</x>
           <y>120</y>
           <width>211</width>
           <height>20</height>
          </rect>
         </property>
        </widget>
        <widget class="QPushButton" name="cambiarNombres_pushButton">
         <property name="geometry">
          <rect>
           <x>330</x>
           <y>240</y>
           <width>141</width>
           <height>41</height>
          </rect>
         </property>
         <property name="text">
          <string>CAMBIAR NOMBRES</string>
         </property>
        </widget>
       </widget>
      </widget>
      <widget class="QWidget" name="tabMoverPorExtension">
       <attribute name="title">
        <string>Mover Extension</string>
       </attribute>
       <widget class="QGroupBox" name="groupBox">
        <property name="geometry">
         <rect>
          <x>70</x>
          <y>40</y>
          <width>531</width>
          <height>351</height>
         </rect>
        </property>
        <property name="title">
         <string>Mover de origen a destino por filtrado de extension</string>
        </property>
        <layout class="QGridLayout" name="gridLayout">
         <item row="0" column="0">
          <widget class="QLabel" name="label">
           <property name="text">
            <string>Carpeta de origen: </string>
           </property>
          </widget>
         </item>
         <item row="0" column="1">
          <widget class="QLineEdit" name="carpetaOrigen"/>
         </item>
         <item row="0" column="2">
          <widget class="QToolButton" name="origenBoton">
           <property name="text">
            <string>...</string>
           </property>
          </widget>
         </item>
         <item row="1" column="0">
          <widget class="QLabel" name="label_2">
           <property name="text">
            <string>Carpeta de destino:</string>
           </property>
          </widget>
         </item>
         <item row="1" column="1">
          <widget class="QLineEdit" name="carpetaDestino"/>
         </item>
         <item row="1" column="2">
          <widget class="QToolButton" name="destinoBoton">
           <property name="text">
            <string>...</string>
           </property>
          </widget>
         </item>
         <item row="2" column="0">
          <widget class="QLabel" name="label_3">
           <property name="text">
            <string>Extensi√≥n del archivo:</string>
           </property>
          </widget>
         </item>
         <item row="2" column="1" colspan="2">
          <widget class="QLineEdit" name="extensionArchivo"/>
         </item>
         <item row="3" column="0" colspan="3">
          <widget class="QPushButton" name="moverArchivosBoton">
           <property name="text">
            <string>MOVER ARCHIVOS</string>
           </property>
          </widget>
         </item>
        </layout>
       </widget>
      </widget>
      <widget class="QWidget" name="tabMoverPorNombres">
       <attribute name="title">
        <string>Mover Nombre</string>
       </attribute>
       <widget class="QGroupBox" name="groupBox_3">
        <property name="geometry">
         <rect>
          <x>70</x>
          <y>40</y>
          <width>531</width>
          <height>351</height>
         </rect>
        </property>
        <property name="title">
         <string>Mover de origen a destino por filtrado de nombre</string>
        </property>
        <layout class="QGridLayout" name="gridLayout_2">
         <item row="0" column="0">
          <widget class="QLabel" name="label_6">
           <property name="text">
            <string>Carpeta de origen: </string>
           </property>
          </widget>
         </item>
         <item row="0" column="1">
          <widget class="QLineEdit" name="rutaOrigen_porNombre_lineEdit"/>
         </item>
         <item row="0" column="2">
          <widget class="QToolButton" name="rutaOrigen_porNombre_toolButton">
           <property name="text">
            <string>...</string>
           </property>
          </widget>
         </item>
         <item row="1" column="0">
          <widget class="QLabel" name="label_7">
           <property name="text">
            <string>Carpeta de destino:</string>
           </property>
          </widget>
         </item>
         <item row="1" column="1">
          <widget class="QLineEdit" name="rutaDestino_porNombre_lineEdit"/>
         </item>
         <item row="1" column="2">
          <widget class="QToolButton" name="rutaDestino_porNombre_toolButton_2">
           <property name="text">
            <string>...</string>
           </property>
          </widget>
         </item>
         <item row="2" column="0">
          <widget class="QLabel" name="label_8">
           <property name="text">
            <string>Nombre a filtrar:</string>
           </property>
          </widget>
         </item>
         <item row="2" column="1" colspan="2">
          <widget class="QLineEdit" name="nombreFiltrar_porNombre_lineEdit_3"/>
         </item>
         <item row="3" column="0" colspan="3">
          <widget class="QPushButton" name="moverPorNombre_pushButton">
           <property name="text">
            <string>MOVER ARCHIVOS</string>
           </property>
          </widget>
         </item>
        </layout>
       </widget>
      </widget>
      <widget class="QWidget" name="tabMoverSeleccionados">
       <attribute name="title">
        <string>Mover Seleccionados</string>
       </attribute>
       <widget class="QGroupBox" name="groupBox_4">
        <property name="geometry">
         <rect>
          <x>9</x>
          <y>9</y>
          <width>998</width>
          <height>523</height>
         </rect>
        </property>
        <property name="title">
         <string>Mover Seleccionados</string>
        </property>
        <widget class="QPlainTextEdit" name="mover_seleccionados_plainTextEdit">
         <property name="geometry">
          <rect>
           <x>30</x>
           <y>50</y>
           <width>311</width>
           <height>431</height>
          </rect>
         </property>
        </widget>
        <widget class="QPushButton" name="mover_seleccionados_pushButton">
         <property name="geometry">
          <rect>
           <x>410</x>
           <y>280</y>
           <width>171</width>
           <height>61</height>
          </rect>
         </property>
         <property name="text">
          <string>MOVER SELECCIONADOS</string>
         </property>
        </widget>
        <widget class="QLabel" name="label_9">
         <property name="geometry">
          <rect>
           <x>390</x>
           <y>180</y>
           <width>61</width>
           <height>16</height>
          </rect>
         </property>
         <property name="text">
          <string>Nueva Ruta:</string>
         </property>
        </widget>
        <widget class="QLineEdit" name="nueva_ruta_mover_seleccionados_lineEdit">
         <property name="geometry">
          <rect>
           <x>470</x>
           <y>180</y>
           <width>181</width>
           <height>20</height>
          </rect>
         </property>
        </widget>
        <widget class="QToolButton" name="elegir_nueva_ruta_mover_seleccionados_toolButton">
         <property name="geometry">
          <rect>
           <x>660</x>
           <y>180</y>
           <width>25</width>
           <height>19</height>
          </rect>
         </property>
         <property name="text">
          <string>...</string>
         </property>
        </widget>
        <widget class="QToolButton" name="elegir_archivos_mover_seleccionados_toolButton_2">
         <property name="geometry">
          <rect>
           <x>530</x>
           <y>80</y>
           <width>25</width>
           <height>19</height>
          </rect>
         </property>
         <property name="text">
          <string>...</string>
         </property>
        </widget>
        <widget class="QLabel" name="label_10">
         <property name="geometry">
          <rect>
           <x>390</x>
           <y>80</y>
           <width>102</width>
           <height>16</height>
          </rect>
         </property>
         <property name="text">
          <string>Seleccionar Archivos:</string>
         </property>
        </widget>
       </widget>
      </widget>
     </widget>
    </item>
   </layout>
  </widget>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>1040</width>
     <height>21</height>
    </rect>
   </property>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
 </widget>
 <resources/>
 <connections/>
</ui>
